cheatsheets_draft/Kubernetes_Security_Cheat_Sheet.md

```diff
@@ -270,6 +270,54 @@ spec:
 
 For more information on security context for Pods, refer to the documentation at https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 
+### Implement Service Mesh

```

Comment:
can you describe some disadvantages of using service mesh?

Reply:
Done, in commit https://github.com/OWASP/CheatSheetSeries/pull/496/commits/a1275b258856778c2e73a641b780e359b1a1b4b7